test/feature/features/multicoordinator.feature

```diff
@@ -18,38 +18,23 @@ Feature: Coordinator test
     """
     router
     """
-    When I run SQL on host "coordinator2"
+    When I run SQL on host "waiting_coordinator"
     """
     SHOW routers
     """
     Then command return code should be "1"
 
   Scenario: Second coordinator turns on when other is dead
-    Given host "coordinator" is stopped
-    When I run SQL on host "coordinator2"
+    Given host "router" is stopped
+    And host "router" is started
+    And host "router2" is stopped
+    And host "router2" is started
+    When I execute SQL on host "coordinator"
     """
     SHOW routers
     """
     Then command return code should be "0"
     And SQL result should match regexp
     """
     router
-    """
-
-    Scenario: first coordinator awaits after recovery
```

Comment:
Why?

Reply:
Because host with coordinator might change during test. After cluster is up or any router is down, new coordinator connection is established. Because maybe it was a router with running coordinator. That's why we cannot check this anymore.
This test assumes we have coordinator on router-1 and coordinator on router-2, and it assumes it remains true after restart of any coordinator. But this is not always true